src/toil_scripts/adam_pipeline/spark_toil_script.py

```diff
@@ -348,6 +362,8 @@ def build_parser():
                         help = 'Amount of memory to allocate for Spark Driver.')
     parser.add_argument('-q', '--executor_memory', required = True,
                         help = 'Amount of memory to allocate per Spark Executor.')
+    parser.add_argument('-r', '--retain_spark_logs', action = 'store_true',
```

Comment:
Add `default=False`, otherwise it would be None by default. None evaluates to False in a boolean context but why make it complicated. You may need this to be tristate in the future.